xbmc/cores/RetroPlayer/process/RPProcessInfo.cpp

```diff
@@ -64,9 +64,9 @@ CRPProcessInfo::CRPProcessInfo(std::string platformName)
 
 CRPProcessInfo::~CRPProcessInfo() = default;
 
-CRPProcessInfo* CRPProcessInfo::CreateInstance()
+std::unique_ptr<CRPProcessInfo> CRPProcessInfo::CreateInstance()
 {
-  CRPProcessInfo* processInfo = nullptr;
+  std::unique_ptr<CRPProcessInfo> processInfo = nullptr;
```

Comment:
Drop the ` = nullptr`.

Reply:
And below change the `processInfo != nullptr` comparison to implicit boolean cast